configs/pulpcore/nightly.yaml

```diff
@@ -0,0 +1,64 @@
+---
+:project: pulpcore
+:github_org: pulp
```

Comment:
not sure what implications this has, but pretty sure it won't live in that org ;-)

Reply:
ah, yeah, :D old habits